README.md

```diff
@@ -198,6 +198,101 @@ To add copyright headers to all files with IntelliJ
 2. Create a new copyright profile matching existing copyright header found on all files, name it Dockstore (Settings -> Copyright -> Copyright Profiles -> Add New)
 3. Set the default project copyright to Dockstore (Settings -> Copyright)
 
+### Setting up a Mac for Dockstore development
+Install Docker (Be sure to click on 'Mac with Apple chip')
```

Comment:
Suggest a message similar to what you have on line 287: "if on Apple Silicon, select the .dmg (Apple Silicon), otherwise suggest .dmg(Intel)" -- whatever the Docker wording is.

Reply:
done